### ISSUE TITLE
[Fix proposed] TabView bottom border line is too short

### ISSUE THREAD

**Issue by eugenegff** [microsoft/microsoft-ui-xaml#5954]
**Describe the bug**
TabView bottom border line is too short in new styles

**Screenshots**
![image](https://user-images.githubusercontent.com/34012295/134601814-b5bf7b41-3416-47a8-9265-65388f4f7188.png)

**Comment by eugenegff** [comment]
Note, that in styles v1 shadow had Grid.ColumnSpan="4", covering LeftContentColumn, TabColumn, AddButtonColumn, RightContentColumn, but in Win11 like styles separator line covers only TabColumn and AddButtonColumn - seems to be simply overlooked

**Comment by beervoley** [comment]
@eugenegff I believe in this case it's caused by having custom content it **TabView Footer**. 
If this is the case, it was done on purpose so users can customize that region without having to be locked in with the provided border. If you need to **extend** the border in this case, please add the border to your custom content :)

**Comment by eugenegff** [comment]
> @eugenegff I believe in this case it's caused by having custom content it **TabView Footer**.
> If this is the case, it was done on purpose so users can customize that region without having to be locked in with the provided border. If you need to **extend** the border in this case, please use **TabViewBorderBrush** for the bottom border of the footer :)

That TabView is extended into TitleBar as in https://docs.microsoft.com/en-us/windows/apps/design/controls/tab-view sample, there are 
```
                    <muxc:TabView.TabStripHeader>
                        <Grid x:Name="ShellTitlebarInset" Background="Transparent" />
                    </muxc:TabView.TabStripHeader>
                    <muxc:TabView.TabStripFooter>
                        <Grid x:Name="CustomDragRegion" Background="Transparent" />
                    </muxc:TabView.TabStripFooter>
```

**Comment by beervoley** [comment]
> > @eugenegff I believe in this case it's caused by having custom content it **TabView Footer**.
> > If this is the case, it was done on purpose so users can customize that region without having to be locked in with the provided border. If you need to **extend** the border in this case, please use **TabViewBorderBrush** for the bottom border of the footer :)
> 
> That TabView is extended into TitleBar as in https://docs.microsoft.com/en-us/windows/apps/design/controls/tab-view sample, there are
> 
> ```
>                     <muxc:TabView.TabStripHeader>
>                         <Grid x:Name="ShellTitlebarInset" Background="Transparent" />
>                     </muxc:TabView.TabStripHeader>
>                     <muxc:TabView.TabStripFooter>
>                         <Grid x:Name="CustomDragRegion" Background="Transparent" />
>                     </muxc:TabView.TabStripFooter>
> ```
> 
> If you insist that such non-intuitive and not backward compatible behavior is by design - then sample should probably be fixed.
> 
> Note, that XamlControlsGallery has many samples of TabView without TabStripFooter - all with this visual glitch. It actually forces us to always create TabStripFooter to fix it - and it is not good. OTOH, (rare?) applications that does not want this stroke at TabStripFooter can always hide it under opaque TabStripFooter or override TabViewBorderBrush
> 
> ![image](https://user-images.githubusercontent.com/34012295/134970300-f34eff2a-3a1e-4fdc-a6fb-0b5876f1bc0e.png)
> 
> Here is screenshot of released app that uses WinUI 2.6 that shows that this visual glitch is regression - WinUI 2.6 has shadow at this place instead of the stroke line, and shadow extends on whole window width
> 
> ![image](https://user-images.githubusercontent.com/34012295/134970913-eebfd550-feea-4f1a-9d29-6834c18763d9.png)

I see now. Yes, you're right - it's not very intuitive and it definitely should span the entire width. :)
